TEM_comms/buffer.py

```diff
@@ -2,6 +2,7 @@
 
 
 class Status(BaseMessage):
+    montage_id: str
```

Comment:
How do you propose to use the `montage_id` field in the status message? The Montage manager may be in different stages of processing multiple montages at once. For example, it may be collecting tiles and metadata of one montage, while uploading a second.

Reply:
I was considering this last night.  Lets go over the status message at 10 before I do anything on this.